src/main/resources/bundles/bundle_vi.properties

```diff
@@ -390,55 +394,33 @@ ui.button.back = [#6ced7d]\uE802 Trở lại
 
 # region stats
 stats.title =  Thông số người chơi
-stats.content = \n[accent][lightgray] Biệt danh: {0}\
-  \n[accent][lightgray] Định danh: {1}\
-  \n\n[accent][lightgray] Thứ hạng: {2}\
-  \n[gray]{3}[accent]\
-  \n\n[#6ced7d][lightgray] Khối đã xây: []{4}\
-  \n[scarlet][lightgray] Khỗi đã phá dỡ: []{5}\
-  \n\n[orange][lightgray] Trò chơi đã chơi: []{6}\
-  \n[orange][lightgray] Số lượt đã vượt qua: []{7}\
-  \n\n[accent]<<< CHIẾN THẮNG >>>\
-  \n[lightgray] Tấn công: []{8}\
-  \n[lightgray] Thành trì: []{9}\
-  \n[lightgray] Pháo đài: []{10}\
-  \n[lightgray] Lục giác: []{11}\
-  \n⚠[lightgray] MS:GO: []{12}\
-  \n[lightgray] PvP: []{13}\
-  \n[lightgray] SPvP: []{14}\
-  \n\n[lightgray] Tổng thời gian chơi: \n[]{15}
-stats.1va = [white](mvg: [accent]{0}[])[]
+stats.content = \n[accent][lightgray] Biệt danh: {0}\n[accent][lightgray] Định danh: {1}\n\n[accent][lightgray] Thứ hạng: {2}\n[gray]{3}[accent]\n\n[#6ced7d][lightgray] Khối đã xây: []{4}\n[scarlet][lightgray] Khỗi đã tháo dỡ: []{5}\n\n[orange][lightgray] Trò chơi đã chơi: []{6}\n[orange][lightgray] Số lượt đã vượt qua: []{7}\n\n[accent]<<< CHIẾN THẮNG >>>\n[lightgray] Tấn công: []{8}\n[lightgray] Thủ thành: []{9}\n[lightgray] Pháo đài: []{10}\n[lightgray] Lục giác: []{11}\n⚠[lightgray] MS:GO: []{12}\n[lightgray] PvP: []{13}\n[lightgray] SPvP: []{14}\n\n[lightgray] Tổng thời gian chơi: \n[]{15}
```

Comment:
Please make newlines here visually distinct. It all being in a single line is hard to read and modify